package.json

```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "async": "^3.2.0",
+    "axios": "^0.21.0",
```

Comment:
Can't we use Request instead of adding an additional dependency? 

I'm all for switching to axios, but not keeping two different HTTP client libraries.

Reply:
Hadn't noticed this. Followed model from existing `blockchainAddressApi.js` and used `request` vs `axios.` Removed `axios.`